build/virt-v2v/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:8
```

Comment:
What do you think about using `registry.access.redhat.com/ubi8/ubi-minimal`. I see that good number of containers in `kubevirt` org is using this base image.

Reply:
Unfortunately, `ubi8` is not able to pull `qemu-guest-agent` package, and others. The `ubi-8-appstream` mainly contains packages related to development, like compilers, libraries..., so nothing related to more common OS usage.